coordinates/EMC_multi_gridCoordinates.m

```diff
@@ -0,0 +1,337 @@
+function [gX, gY, gZ, vX, vY, vZ] = EMC_multi_gridCoordinates(SIZE, METHOD, TRANS, ...
+                                                              flgNormalize, ...
+                                                              varargin)
+% [gX, gY, gZ, vX, vY, vZ] = EMC_multi_gridCoordinates(SIZE, METHOD, TRANS, flgNormalize, varargin)
+%
+% Compute flow-field grids (gridCoordinates), for interpolations.
+% WARNING: To compute grid masks, use EMC_multi_gridMasks.
+% WARNING: To compute gridVectors, use EMC_multi_gridVectors.
+%
+% SIZE (vector):                Size of the grid to compute (x, y, z) or (x, y).
+%                               Sould be a 2d/3d row vector of integers.
+%
+% METHOD (str):                 Device to use; 'gpu' or 'cpu'.
```

Comment:
I capitalize GPU and lowercase cpu because it is much easier to see the difference when you are scanning the code.